src/components/CardSelect/components/CardAdditionalInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import { Divider } from '../../Divider'
+import { Text } from '../../Text'
+import React, { ReactElement } from 'react'
+import { TInfoList } from '../types'
+
+interface TCardAdditionalInfo {
+  additionalInfo?: TInfoList
+  disabled?: boolean
+}
+export const CardAdditionalInfo = (props: TCardAdditionalInfo): ReactElement | null => {
+  const { additionalInfo, disabled } = props
+  return additionalInfo?.length ? (
+    <>
+      <Divider type={'primary'} isHorizontal={true} className={'mt-16 mb-16'} />
+      <div className="card-select__list">
```

Comment:
in some cases you are writing className="card-select__list" in other ones className={"card-select__list"}.  Lets have a consistency for all attributes that is string